kubernetes/clientset.go

```diff
@@ -1,63 +0,0 @@
-//   Copyright 2016 Wercker Holding BV
-//
-//   Licensed under the Apache License, Version 2.0 (the "License");
-//   you may not use this file except in compliance with the License.
-//   You may obtain a copy of the License at
-//
-//       http://www.apache.org/licenses/LICENSE-2.0
-//
-//   Unless required by applicable law or agreed to in writing, software
-//   distributed under the License is distributed on an "AS IS" BASIS,
-//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-//   See the License for the specific language governing permissions and
-//   limitations under the License.
-
-package kubernetes
-
-import (
-	"path/filepath"
-
-	"github.com/pkg/errors"
-	"k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/tools/clientcmd"
-
-	// load all auth plugins
-	_ "k8s.io/client-go/plugin/pkg/client/auth"
```

Comment:
I think we need to move this import to another file like main.go.

```
$ dist/stern
panic: no Auth Provider found for name "oidc"

goroutine 1 [running]:
k8s.io/client-go/kubernetes.NewForConfigOrDie(...)
        /go/pkg/mod/k8s.io/client-go@v0.28.0/kubernetes/clientset.go:708
github.com/stern/stern/cmd.(*options).Complete(0xc00041d200, {0x34f05e0?, 0x0?, 0x0?})
        /go/src/github.com/stern/stern/cmd/cmd.go:139 +0x295
github.com/stern/stern/cmd.NewSternCmd.func1(0xc0003aa600, {0x34f05e0?, 0x4?, 0x242d247?})
        /go/src/github.com/stern/stern/cmd/cmd.go:648 +0x54
github.com/spf13/cobra.(*Command).execute(0xc0003aa600, {0xc0000400a0, 0x0, 0x0})
        /go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:940 +0x87c
github.com/spf13/cobra.(*Command).ExecuteC(0xc0003aa600)
        /go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3a5
github.com/spf13/cobra.(*Command).Execute(0x0?)
        /go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992 +0x13
main.main()
        /go/src/github.com/stern/stern/main.go:32 +0xaf
```

Reply:
nice catch! I've fixed it.